### PR TITLE
Updated symfony requirements for 5.4.

### PR DESCRIPTION
**Pull request by Raistlfiren**
Just updated Symfony requirements to be compatible with Symfony 5.4.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,9 +25,9 @@
     },
     "require": {
         "php": ">=5.5.9",
-        "symfony/framework-bundle": "^3.4",
-        "symfony/twig-bundle": "^3.4",
```
Comment (bocharsky-bw) [inline composer.json:29]: I think we still want to support Symfony 4.4 which is LTS and supports bug fixes until Nov 2022. And also, there's Symfony 6 already, we need to allow it as well :)

I'd write constraints as `^4.4|^5.4|^6.0`. Could you make this change for both Symfony dependencies, please?

File: composer.json [hunk 2]
```diff
@@ -25,9 +25,9 @@
...
-        "twig/twig": "^2.0"
```
Comment (bocharsky-bw) [inline composer.json:30]: And if we want to support legacy versions I mentioned above - bumping twig to the new major version might be too aggressive. I'd rather allow Twig 2.0 or Twig 3.0, i.e. `^2.0|^3.0`

**Review by bocharsky-bw — CHANGES_REQUESTED** [review]
Hey @Raistlfiren ! Thank you for creating this PR! Yeah, we definitely should allow Symfony 5.4, but in a bit different way. Could you make those changes I mentioned?

**Comment by Raistlfiren** [comment]
Thank you @bocharsky-bw for the feedback. I thought there was a better way than the approach I took, but I wasn't sure of how to do it. I updated the `composer.json` as you requested. Thank you for maintaining and taking care of this repository.

**Comment by bocharsky-bw** [comment]
Nice, I'm happy now :)

And thank you for your interest in this bundle! I haven't used this one since Symfony 3.4 probably, as you can see from the old composer.json 😇  But I think this bundle is still good for this simple purpose :)

Cheers!

**Pull request merged**